src/key.js

```diff
@@ -1769,3 +1773,27 @@ export async function isAeadSupported(keys, date = new Date(), userIds = []) {
   }));
   return supported;
 }
+
+async function bindSignature(subkey, primaryKey, options) {
```

Comment:
Perhaps name this `createBindingSignature`?

Also, bit nitpicky but could you put this function a bit closer to the rest of the related code? E.g. under `wrapKeyObject`.

Reply:
I want to refactor this module anyway, let it stay here for now, that way file diff is minimised.